hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1507,64 +1507,79 @@
       name: chip_otbn_op
       desc: '''Verify an OTBN operation.
 
-            SW test directs the BIGNUM engine to perform an operation. The BIGNUM SW image is
-            backdoor loaded into OTBN IMEM. SW validates the reception of the otbn done interrupt.
-            SW also verifies the correctness of the OTBN operation using a reference model. Details
-            TBD.
+            - SW test directs the OTBN engine to perform an ECDSA operation.
+            - SW validates the reception of the otbn done interrupt once the operation is complete.
+            - SW also verifies the correctness of the result using the reference model generated
+              result.
             '''
       milestone: V2
       tests: []
     }
     {
-      name: chip_otbn_imem_uncorrectable_alert
-      desc: '''Verify the imem uncorrectable alert from OTBN.
+      name: chip_otbn_rnd_entropy
+      desc: '''Verify OTBN can fetch RND numbers from the entropy src.
 
-            Inject 2 bit errors within the IMEM SRAM inside OTBN via backdoor and ensure that this
-            alert propagates to an NMI.
+            - SW initializes the entropy subsystem to generate randomness.
+            - SW loads an OTBN app that executes instructions to read the RND bits.
+            - The OTBN app ensures that the values when read consequtively do not match, and its not
+              all 0s or all 1s, as a basic measure to ensure that the entropy subsystem is returning
+              some data.
             '''
       milestone: V2
       tests: []
     }
     {
-      name: chip_otbn_dmem_uncorrectable_alert
-      desc: '''Verify the dmem uncorrectable alert from OTBN.
+      name: chip_otbn_urnd_entropy
+      desc: '''Verify OTBN can fetch URND numbers from the entropy src.
 
-            Inject 2 bit errors within the DMEM SRAM inside OTBN via backdoor and ensure that this
-            alert propagates to an NMI.
+            - Similar to chip_otbn_rnd_entropy, but verifies the URND bits.
             '''
       milestone: V2
       tests: []
     }
     {
-      name: chip_otbn_reg_uncorrectable_alert
-      desc: '''Verify the reg uncorrectable alert from OTBN.
+      name: chip_otbn_idle
+      desc: '''Verify the OTBN idle signal to clkmgr.
 
-            Details TBD. Ensure that this alert propagates to an NMI.
+            - Write the OTBN clk hint to 1 within clkmgr to indicate OTBN clk is ready to be gated.
+              Verify that the OTBN clk hint status within clkmgr reads 0 (OTBN is idle).
+            - Start an OTBN operation.
+              Verify that the OTBN clk hint status within clkmgr now reads 1 (OTBN is not idle),
+              before the operation is complete.
+            - After the OTBN operation is complete, check the output for correctness.
+              Verify that the OTBN clk hint status within clkmgr now reads 0 again (OTBN is idle),
+              after the operation is complete.
```

Comment:
This might be a silly question, but wouldn't OTBN's clock be gated after the first step here? How would we start it in this case? Maybe I've misunderstood about how clkmgr works, but I assumed this test would be something like "start a long running OTBN job; tell clkmgr to gate it; wait a bit; check that OTBN completed successfully without being disturbed; check that OTBN is now clock-gated" ?

Reply:
No, that's just the 'hint' that we are 'ok to gate the otbn clock when the pwrmgr instructs the clkmgr to do so'. Please see clkmgr design doc.

Reply:
Oh, ok, now I understand. Sounds good.